Monika After Story/game/script-topics.rpy

```diff
@@ -6840,12 +6840,35 @@ init 5 python:
 label monika_playerapologizes:
     # if there's no reason to apologize
     if mas_apology_reason is None:
-        m 1ekd "Did something happen?"
-        m 2ekc "I see no reason for you to be sorry."
-        m 1dsc "..."
-        m 1eub "Anyway, thank you for the apology."
-        m 1eua "Whatever it is, I know you're doing your best to make things right."
-        m 1hub "That's why I love you, [player]!"
+        if mas_isMoniBroken():
+            m 1ekc "... Oh..."
+            m 2rksdlc "You...{w=1.5} want to apologize?"
+            m 2dsc ".{w=1.5}.{w=1.5}."
+            m ".{w=2}.{w=2}."
+            m 1ektsd "I hope so..." 
+            m "{fast}I don't want to give up...{nw}"
+            $ _history_list.pop()
+        elif mas_isMoniDis():
+            m 2lfc "{i}sigh{/i}"
```

Comment:
As an asighde, you could optionally `{w=0.5}{nw}` this to make it possibly a little more natural there.
```suggestion
            m 2dfd "{i}sigh{/i}"
```